libs/sdm/src/image/task/mod.rs

```diff
@@ -153,9 +153,14 @@ impl Default for Status {
 
 #[derive(Debug, PartialEq, Eq)]
 pub enum ContainerState {
+    Empty,
```

Comment:
What is the empty state? Would this be the equivalent to the previous `NotFound` ?

Reply:
Possibly similar to `NotFound`, but I wouldn't be able to say. No explanation about `Empty` in https://docs.rs/bollard/latest/bollard/models/enum.ContainerStateStatusEnum.html. I also did not pick it up in the log files.

Reply:
Actually, your questions made me rethink my shutdown proposal, and `ContainerState::NotFound` should not be removed. I expanded the Docker status mapping and now properly handle `ContainerState::NotFound` during shutdown. I could also remove the sleep-when-exit statements in the launchpad front end.

When the containers are running `fn inspect_container` will return some status, but when they are removed, for example during shutdown, it will return an error, which we then map to  `ContainerState::NotFound`.